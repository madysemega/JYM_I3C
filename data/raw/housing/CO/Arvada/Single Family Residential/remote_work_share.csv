predictions_remote_work_share
7.3707355594635
3.169679435491562
7.073269755840301
4.639537434577941
2.8068275582790374
5.936922299861908
5.2130468916893005
2.5388806676864624
6.223268716335296
8.081103024482726
1.2765680384635925
6.3282333898544305
6.689325802326202
1.1100614669919013
6.712540066242218
5.227449100017547
4.228937132358551
6.0606406617164605
5.210497081279755
0.39573450952768324
5.488486537933349
6.309163961410522
5.253844256401061
6.224920678138733
7.139158306121826
5.078401453495025
6.649640495777129
