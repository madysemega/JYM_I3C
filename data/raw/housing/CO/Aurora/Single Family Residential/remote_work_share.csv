predictions_remote_work_share
6.44756343126297
7.487325093746185
6.94112884759903
8.117213079929352
7.113119385242462
7.24249558210373
6.368474040031433
7.618449184894562
8.028516862392426
6.8739629960060125
8.552628049850464
7.71033323764801
6.6174717521667485
7.584729704856873
6.3631879782676695
7.8921340203285215
7.204719970226288
9.24419429063797
6.638541600704193
7.288522837162018
8.791522631645202
6.538034853935242
8.216626451015472
5.891683135032654
6.6927150225639345
7.281769523620605
6.617299411296845
