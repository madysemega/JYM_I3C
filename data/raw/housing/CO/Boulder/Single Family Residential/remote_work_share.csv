predictions_remote_work_share
16.616369321346284
16.561135499477388
19.407429206371308
16.75621195435524
16.850728807449343
15.187770565748217
14.135605098009112
16.16195048809052
15.823447068929674
14.20022015810013
12.763136763572694
15.226636285781861
15.87144818544388
15.205084848403933
15.76719062805176
17.27546483278275
16.5636806845665
15.860075422525409
14.149157382249834
14.191293417215348
15.637739456892016
14.285054152011872
13.85846295118332
14.689775767326356
15.885122357606889
15.824942778348925
15.299333134889604
