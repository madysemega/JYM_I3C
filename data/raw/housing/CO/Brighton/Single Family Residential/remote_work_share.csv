predictions_remote_work_share
5.144247166514396
5.421544978022575
7.810823208689689
7.52483413875103
6.711607734560966
9.017956874370574
6.375725943446159
8.606681535243988
7.442137808203697
7.701960114240646
8.821688529253006
9.126697524189948
5.968382694125175
9.33179227888584
6.946134250760078
8.572321954965592
9.113195115923881
6.048314779400825
10.368260078430175
7.487219379544258
6.472395978569984
8.789431877732277
9.712187891006469
8.11969209730625
7.54370562851429
10.20478494644165
10.294451379776
