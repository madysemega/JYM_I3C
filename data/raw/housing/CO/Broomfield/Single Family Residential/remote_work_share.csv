predictions_remote_work_share
15.245917429924013
9.222839739322664
19.177507122755056
13.493794091939929
7.362107607126237
16.14304397940636
11.157180886864664
12.242824163436891
19.591015418767935
11.250997140407565
11.12027773499489
11.66109224498272
10.98773264527321
9.573630770444872
13.719348903894426
11.688627775907518
9.156454750299455
11.542253076434138
9.497259717583658
11.465211447477342
11.510557111501695
10.427233251333238
15.160429199934008
11.944926866292954
13.164469673633578
13.103400226831438
11.029931917190554
