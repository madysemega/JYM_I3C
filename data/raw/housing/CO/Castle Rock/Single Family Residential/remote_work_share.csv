predictions_remote_work_share
4.745512375831604
5.535017471313476
5.919821090698242
3.8985430908203123
4.328105292320251
2.9552489900588985
4.101315498352051
3.629987196922302
4.37941659450531
3.9029991436004634
3.454546284675598
4.460241713523865
5.138000502586364
4.807479677200317
5.43378327846527
4.390935230255127
4.291130752563476
3.926787667274475
3.8510043001174923
3.2679801368713375
4.977146205902099
4.824731731414794
4.2092334318161
4.676775002479553
4.983445687294006
5.4144172906875605
5.025896005630493
