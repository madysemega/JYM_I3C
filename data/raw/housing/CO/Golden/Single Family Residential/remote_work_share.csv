predictions_remote_work_share
8.74628727197647
9.309602269530297
11.202686736583708
10.449635030627249
9.379569501280784
11.516792018413541
9.609443222880364
9.823328261971474
11.142545741796493
9.715706079006194
8.807595862150192
11.525557587146757
10.640685591101645
10.697082291841506
10.55903318762779
10.740689521431921
11.85370842695236
11.351047551631925
10.507322846055029
9.082701614499092
10.708452212810515
10.925788418054578
10.790798413753508
10.113179515004157
9.952469227910042
12.583330180644987
11.387985353469848
